### ISSUE TITLE
File drop feature request

### ISSUE THREAD

**Issue by BaderEddineOuaich** [gammasoft71/xtd#5]
Hello @gammasoft71, thanks again for the work your are doing in this repo, such a steady progress that i appreciate!
i  have a feature request which can be useful to the xtd forms users.
its when a user drops a file into a form and an event will be dispatched serving the files paths dropped, may be something like this:
```cpp
file_drop += [&](control& sender, const file_drop_event_args& e) {
      std::filesytem::path files = e.files();
      ...
    };
```
i was able to achieve this in an example using GLFW's glfwSetDropCallback function. i hope this helps.
you may already have this feature in your future plans, i just want to let u know anyways 😁.
Thanks again! 
   Have a good day 🙌.

**Comment by gammasoft71** [comment]
yes of course, it's in the pipe. But I don't know yet when the drag and drop will be implemented. But I hope in the near future.
